test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -8,7 +8,6 @@
 import os
 
 import numpy as np
-import unittest
 import torch
 import torch.distributed as dist
 from typing import List, Any, Type, cast
```

Comment:
nit: can we directly `from torch.testing._internal.common_utils import run_tests` here and clean up the section of duplicate `from`s